assets/aws/files/bin/teleport-generate-config

```diff
@@ -81,6 +81,38 @@ write_https_keypairs_section() {
 EOS
 }
 
+write_database_section() {
+    # use the value of the variable name which was passed in
+    # this is done because there are two different ways of getting the external hostname
+    # depending on how the AMI is being used and we should support both
+    EXTERNAL_HOSTNAME="${!1}"
+    if [[ "${USE_ACM}" == "true" && "${TELEPORT_PROXY_SERVER_LB}" != "" ]]; then
+        # if an alias for the ACM NLB is configured, use that for proxy's database public_addrs
+        if [[ "${TELEPORT_PROXY_SERVER_NLB_ALIAS}" != "" ]]; then
+            MYSQL_PUBLIC_ADDR="'${TELEPORT_PROXY_SERVER_NLB_ALIAS}:3036'"
+            POSTGRES_PUBLIC_ADDR="'${TELEPORT_PROXY_SERVER_NLB_ALIAS}:5432'"
+            MONGO_PUBLIC_ADDR="'${TELEPORT_PROXY_SERVER_NLB_ALIAS}:27017'"
+        else
+            MYSQL_PUBLIC_ADDR="'${TELEPORT_PROXY_SERVER_LB}:3036'"
+            POSTGRES_PUBLIC_ADDR="'${TELEPORT_PROXY_SERVER_LB}:5432'"
+            MONGO_PUBLIC_ADDR="'${TELEPORT_PROXY_SERVER_LB}:27017'"
+        fi
+    else
+        MYSQL_PUBLIC_ADDR="'${EXTERNAL_HOSTNAME:-$PUBLIC_IP}:3036'"
+        POSTGRES_PUBLIC_ADDR="'${EXTERNAL_HOSTNAME:-$PUBLIC_IP}:5432'"
+        MONGO_PUBLIC_ADDR="'${EXTERNAL_HOSTNAME:-$PUBLIC_IP}:27017'"
+    fi
+    # set up the database listeners and public addresses
+    cat << EOS >> ${USE_CONFIG_PATH}
+  mysql_listen_addr: "0.0.0.0:3036"
+  postgres_listen_addr: "0.0.0.0:5432"
+  mongo_listen_addr: "0.0.0.0:27017"
+  mysql_public_addr: [${MYSQL_PUBLIC_ADDR}]
+  postgres_public_addr: [${POSTGRES_PUBLIC_ADDR}]
```

Comment:
`postgres_public_addr` is already written elsewhere in the script that generates this config file: https://github.com/gravitational/teleport/blob/master/assets/aws/files/bin/teleport-generate-config#L145-L149

Reply:
I will change the logic to not include

> The code changes looks good to me. @greedy52 Could you also take a look.
> 
> @stevenGravy Have you tested this changes with db access flow ? If not I can give it a try

I have tested them for the ha and starter cluster